spec/05-migration/plugins/acme/migrations/003_350_to_360_spec.lua

```diff
@@ -0,0 +1,70 @@
+
+local cjson = require "cjson"
+local uh = require "spec.upgrade_helpers"
+
+if uh.database_type() == 'postgres' then
+    describe("acme plugin migration", function()
+        lazy_setup(function()
+            assert(uh.start_kong())
+        end)
+
+        lazy_teardown(function ()
+            assert(uh.stop_kong(nil, true))
+        end)
+
+        uh.setup(function ()
+            local admin_client = assert(uh.admin_client())
+
+            local res = assert(admin_client:send {
+                method = "POST",
+                path = "/plugins/",
+                body = {
+                    name = "acme",
+                    config = {
+                        account_email = "test@example.com",
+                        storage = "redis",
+                        storage_config = {
+                            redis = {
+                                host = "localhost",
+                                port = 57198,
+                                auth = "secret",
+                                database = 2
```

Comment:
That's actually old schema. The differentiating factor is that it uses: `storage_config.redis.auth` instead of `storage_config.redis.password`. Unfortunately I'm unable to test `namespace` and `scan_count` since migration tests run with OLD_CONTAINER and NEW_CONTAINER. The OLD_CONTAINER is locked at version 2.8.x.x [(link)](https://github.com/Kong/kong-ee/blob/76a8dc76b34c096fb49a4492d6ae7864636c5560/scripts/upgrade-tests/test-upgrade-path.sh#L37) and in that version these are only the fields available for acme.